src/test/java/com/emc/object/s3/Sdk238V4Test.java

```diff
@@ -15,12 +15,17 @@
     @Override
     @Test
     public void testTrailingSlash() throws Exception {
+        S3Config s3Config = AbstractS3ClientTest.s3ConfigFromProperties();
         Sdk238V4Test.TestClient client = new Sdk238V4Test.TestClient(AbstractS3ClientTest.s3ConfigFromProperties());
 
         String bucket = "test-trailing-slash-v4";
         client.createBucket(bucket);
         try {
-            Assert.assertEquals("/" + bucket, client.getLastUri().getPath());
+            if (s3Config.isUseVHost()) {
```

Comment:
Could use client.getS3Config() to get S3Config here.

Reply:
Good catch, missed to port the change in line#19. Will keep the code as Sdk238Test.testTrailingSlash().